test/runtests.jl

```diff
@@ -156,5 +154,18 @@ end
     @test_throws ArgumentError svmtrain(rand(2, 5), ones(5); bad_params...)
 end
 
+@testset "Decision values" begin
+    X = [-2 -1 -1 1 1 2;
+         -1 -1 -2 1 2 1]
+    y = [1, 1, 1, 2, 2, 2]
+    d = [1.5 1.0 1.5 -1.0 -1.5 -1.5;
+         0.0 0.0 0.0  0.0  0.0  0.0]
+
+    model = svmtrain(X, y, kernel=Kernel.Linear)
```

Comment:
```suggestion
    model = svmtrain(X, y, kernel = Kernel.Linear)
```

Reply:
Spaces are preferred here? It is a named argument, not a variable. I have usually seen no spaces around `=` (e.g., [dict.jl](https://github.com/JuliaLang/julia/blob/master/base/dict.jl#L716))

Reply:
oh, the default style of JuliaFormatter and DocumentFormat adds spaces to the keyword arguments.

Maybe we can setup a JuliaFormatter GitHub action for this repo; pick a coding style you like later.